plugins/clipboard/lang/en.js

```diff
@@ -11,5 +11,6 @@ CKEDITOR.plugins.setLang( 'clipboard', 'en', {
 	pasteNotification: 'Press %1 to paste. Your browser doesn‘t support pasting with the toolbar button or context menu option.',
 	pasteArea: 'Paste Area',
 	pasteMsg: 'Paste your content inside the area below and press OK.',
-	fileFormatNotSupportedNotification: 'This file format is not supported. You can try with one of the supported formats: ${formats}.'
+	fileFormatNotSupportedNotification: 'File format ${format} is not supported.',
```

Comment:
I wonder if we can't bold the format :thinking: It would make the notification a little bit more readable.

However, I'm not sure if the bold should be here or in the code that replaces the format in the message.

Reply:
We shouldn't place HTML inside language files as it may confuse language translators. I've changed that to italic as it just looks better from my perspective, but that's subjective so we can discuss that further.